README

```diff
@@ -0,0 +1,55 @@
+# About Check
```

Comment:
Did not realize it until now, but your commit is adding a lot of files. Most of them are probably generated at build time and should not be commited.